flake.nix

```
@@ -0,0 +1,35 @@
+{
+  description = "A devShell example";
```

The description from the repo maybe more useful:

```suggestion
  description = "A new FRAME-based Substrate node, ready for hacking.";
```

---

Ah nice this one slipped me by :+1: